gc/gc-iceberg-inttest/build.gradle.kts

```diff
@@ -22,12 +22,17 @@ plugins {
 
 extra["maven.name"] = "Nessie - GC - Integration tests"
 
-val sparkScala = useSparkScalaVersionsForProject("3.4", "2.12")
+val sparkScala = useSparkScalaVersionsForProject("3.5", "2.12")
 
 dependencies {
   implementation(libs.hadoop.client)
 
   implementation(platform(libs.iceberg.bom))
+
+  // Enforce a single version of Netty among dependencies
+  // (Spark, Hadoop and Azure)
+  implementation(platform("io.netty:netty-bom:4.1.110.Final"))
```

Comment:
Renovate will bump this one. You'd need a version range or make the version explicit in another way using "Gradle magic" (IIRC nessie-client built script has a few of those).

Reply:
I think it's OK to bump this version, as long as one single version is used. If it ever happens that a new Netty version becomes incompatible with one of the components using it, then we'd need to force it – but for now, I think that's fine?

Reply:
Oh - then maybe just add it to `gradle/libs.versions.toml`.
Maybe also make it an `enforcedPlatform`, which _enforces_ the exact versions of that bom, whereas versions from a "plain" `platform` can be bumped.